frontend/src/components/Movies/Movies/Movies.js

```diff
@@ -59,13 +37,9 @@ const Movies = ({
           {searchedMovies.map((movie) => (

```

Comment:
1) I believe you don't need to wrap that in Fragment
2) Don't create map callback as anonymous function right in return statement. It'll lead to the situation when on every render of the component, new function will be created. The same one function. over and over again. You don't need to create it again and again. you need just use it, right? So please, create such callbacks IN functional component. And don't forget useCallback